### PR TITLE
FB-96 Remove forking from CLI manager

### LINKED ISSUE #2585: Remove forking from CLI manager
Part of https://github.com/cargomedia/cm/issues/2524

Please make sure that the CLI exits with non-0 code on failure, to fix https://github.com/cargomedia/cm/issues/2583

### PR DESCRIPTION
**Pull request by alexispeter**
closes #2585

- Remove forking from cli manager
- Use exit-code 1 on failure
- Don't track cli commands in newrelic

### CODE REVIEW

**Comment by njam** [comment]
@zazabe could you review this?

**Comment by zazabe** [comment]
ok

File: library/CM/Cli/CommandManager.php [hunk 1]
```diff
@@ -272,47 +247,41 @@ protected function _getProcess() {
...
+            $this->_outputError('ERROR: ' . $ex->getMessage() . PHP_EOL);
+            if (isset($command)) {
+                $this->_outputError('Usage: ' . $arguments->getScriptName() . ' ' . $command->getHelp());
+            } else {
+                $this->_outputError($this->getHelp());
             }
-        };
+            return false;
+        } catch (Exception $ex) {
+            $this->_outputError('ERROR: ' . $ex->getMessage() . PHP_EOL);
```
Comment (zazabe) [inline library/CM/Cli/CommandManager.php]: Could be nice to have the stacktrace, what about using `CM_ExceptionHandling_Formatter_Plain`?

Comment (alexispeter) [inline library/CM/Cli/CommandManager.php]: Good point.

**Comment by alexispeter** [comment]
please review @zazabe 

The last commit also changes the way invalid packages/commands are handled, showing the exception-message and displaying the help as it would for invalid command-parameters instead of throwing an uncaught exception.

**Pull request merged**